packages/generator-one-app-module/generators/app/templates/base-module/src/components/ModuleContainer.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ModuleRoute from 'holocron-module-route';
+
+const <%=moduleNamePascal%> = () => (
+  <div>
+    <h1>Welcome to One App!</h1>
+  </div>
+);
+
+<%=moduleNamePascal%>.childRoutes = () => ([
+  <ModuleRoute path="/" />,
+]);
+
+if (!global.BROWSER) {
```

Comment:
I think a comment linking to the explanation for why this is wrapped in this if statement is needed. There is a good explanation in the one app readme we can add as a link.

Reply:
great idea. Linked it.